.rollup/index.mjs

```diff
@@ -39,7 +39,7 @@ function createRollupConfig (options = defaultOptions()) {
         format: 'esm'
       }
     ],
-    external: ['@bugsnag/core'].concat(options.external),
+    external: options.external,
```

Comment:
the notifiers themselves use this `createRollupConfig` function but we need `@bugsnag/core'` bundling in those. Its more code but I think clearer to be explicit about this.

Reply:
This is apparent now because previously the imports of `@bugsnag/core/breadcrumb` etc weren't being treated as external, and the notifiers we not importing from `@bugsnag/core` until now